wellregistry/wellregistry/settings.py

```diff
@@ -175,10 +178,17 @@
 
 if 'test' in sys.argv:
```

Comment:
There is no need for this if statement anymore. The database is configured the same way regardless of whether tests are running are not.

Reply:
Actually I was wrong. You do need the if statement. We need to use the Database_username for testing so that the tests can create the test database. So please leave in.